### PR TITLE
refactor: replace touchid with identity

### PR DESCRIPTION
**Pull request by ewanharris**
touchid is being removed from the sdk

Looking at the identity repo the apiname is fixed, so I figured I'd enable that 🤞

### CODE REVIEW

File: Resources/require.test.js [hunk 1]
```diff
@@ -200,21 +200,21 @@ describe('require()', function () {
 		should(abbrev('foo', 'fool', 'folding', 'flop')).eql({ fl: 'flop', flo: 'flop', flop: 'flop', fol: 'folding', fold: 'folding', foldi: 'folding', foldin: 'folding', folding: 'folding', foo: 'foo', fool: 'fool' });
 	});
 
-	// FIXME We have no native ti.touchid module for windows!
+	// FIXME We have no native ti.identity module for windows!
 	it.windowsMissing('loads native module by id', function () {
-		var object = require('ti.touchid');
+		var object = require('ti.identity');
 		should(object).have.property('apiName');
 		// Of course, the module's apiName is wrong, so we can't test that
```
Comment (sgtcoolguy) [inline Resources/require.test.js]: drop the comment too

File: Resources/ti.touchid/example.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
```
Comment (sgtcoolguy) [inline Resources/ti.touchid/example.js]: this filename needs to be changed (not just the internal contents)

Comment (ewanharris) [reply Resources/ti.touchid/example.js]: doi, of course..

File: Resources/require.test.js (hunk 1 above)
Comment (ewanharris) [reply Resources/require.test.js]: iOS looks to be pulling `Ti.Proxy` here so I'll revert adding this back in

**Comment by sgtcoolguy** [comment]
merged in manually.

**Pull request closed without merging**